### ISSUE TITLE
Add option to export textures

### ISSUE THREAD

**Issue by thomthom** [thomthom/cities-skylines-sketchup-tools#1]
Add option to export textures.

For diffuse the model material can be exported.

For the others a webdialog with options for each type supported can be added. Save the export preference with the model.

**Comment by Mistro11** [comment]
Hi ThomThom. Great work.
The test building I used was exported with a blue diffuse color. I chose a 4x4 industrial building as the template. It successfully imported into the asset editor with no problem but the texture was a desaturated gold color with what looks like some graphics on the ground plane. Is this expected or should it come in blue as I had it in Sketchup? Using SU2015 Pro BTW.

**Comment by thomthom** [comment]
That sounds like the color variation features of the came. Expand the Materials section in the Properties window:

![2015-04-18_11h12_36](https://cloud.githubusercontent.com/assets/192418/7214804/e7a2c3ac-e5bb-11e4-96e9-f8a26c3bb18a.png)

**Comment by Mistro11** [comment]
You're right about the color variation. I played with it a bit. I'm still learning the editor properties. Are you writing a script that creates 1 texture map with UV coordinates for the model?

**Comment by thomthom** [comment]
No - that's UV unwrapping and that's very difficult to do auto-magically. This issue was an idea I had where the exporter would open a dialog where you have multiple file fields where you could pick textures for the differente texture channels the game support and it'd copy them to the game asset folder.

Though - don't expect this to happen immediately. I shouldn't really have been doing this exporter in the first place - I have too many other things I should have done first.
